test/include/tests/test_utils/executor_shutdowner.h

```diff
@@ -1,7 +1,7 @@
 #ifndef CONCURRENCPP_EXECUTOR_TEST_HELPERS_H
 #define CONCURRENCPP_EXECUTOR_TEST_HELPERS_H
 
-#include "../../concurrencpp/src/executors/executor.h"
+#include "concurrencpp/concurrencpp.h"
```

Comment:
This should probably include only what's necessary/was previously there.
```suggestion
#include "concurrencpp/executors/executor.h"
```